.github/workflows/prod-chat-CI.yml

```
@@ -0,0 +1,35 @@
+
+      # Secret 파일 주입 (CHAT)
+      - name: Inject CHAT secrets
+        run: |
+          echo "${{ secrets.CHAT_APPLICATION_PROD }}" > chat-server/src/main/resources/application-chat-prod.yml
+          mkdir -p chat-server/src/main/resources/firebase
+          echo "${{ secrets.CHAT_FIREBASE_JSON }}" > chat-server/src/main/resources/firebase/napzak-firebase-adminsdk.json
+
+      - name: Build (Gradle)
+        run: |
+          chmod +x ./gradlew
+          ./gradlew clean build -x test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)